packages/editor-ui/src/components/ExpressionEditorModal/n8nLanguagePack/index.cjs

```diff
@@ -0,0 +1,57 @@
+'use strict';
```

Comment:
Everything under `/n8nLanguagePack` is auto-generated externally. Will decide whether to include or publish in next phase.